sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationAsyncClient.java

```diff
@@ -81,8 +81,8 @@
      * {@codesnippet com.azure.data.appconfiguration.configurationasyncclient.addConfigurationSetting#string-string-string}
      *
      * @param key The key of the configuration setting to add.
-     * @param label The label of the configuration setting to add, or optionally, null if a setting with
-     * label is desired.
+     * @param label The label of the configuration setting to add, or optionally, null if the label is not desired in
```

Comment:
Let's clean up the wording a little, use the positive case instead of the negative.

```suggestion
     * @param label The label of the configuration setting to add. If {@code null} no label will be used.
```

If we are happy with this change let's use it for the other Javadocs as well.